src/recordlinker/schemas/pii.py

```diff
@@ -163,6 +165,25 @@ class Address(pydantic.BaseModel):
     latitude: typing.Optional[float] = None
     longitude: typing.Optional[float] = None
 
+    @functools.cached_property
+    def normalize_state(self) -> str | None:
+        """
+        Normalize the state field into 2-digit USPS code.
+        """
+
+        _STATE_NAME_TO_CODE = utils.read_json("assets/states.json")
```

Comment:
Can we move these to two variables to either the module or class level.  As is, it will call this every time the function is invoked, but we really only need to call it once.